javalanglib/src/main/scala/java/lang/Math.scala

```diff
@@ -305,13 +305,35 @@ object Math {
     if (a >= Integer.MIN_VALUE && a <= Integer.MAX_VALUE) a.toInt
     else throw new ArithmeticException("Integer overflow")
 
+  def floorDiv(a: scala.Int, b: scala.Int): scala.Int = {
+    if (b == 0)
+      throw new ArithmeticException("/ by 0")
```

Comment:
Don't explicitly throw `ArithmeticException`s here. Let it be as undefined behavior as `a / b` itself.